src/ert/cli/workflow.py

```diff
@@ -20,7 +20,12 @@ def execute_workflow(
         msg = "Workflow {} is not in the list of available workflows"
         logger.error(msg.format(workflow_name))
         return
-    runner = WorkflowRunner(workflow=workflow, storage=storage, ert_config=ert_config)
+    runner = WorkflowRunner(
+        workflow=workflow,
+        storage=storage,
+        ert_config=ert_config,
```

Comment:
I guess we should remove this as it is not available in run models, where most of these are run from?